Jenkinsfile.ci

```diff
@@ -554,11 +555,10 @@ def linuxVcxBuild(file, env_name, network_name) {
         testEnv.inside("--network=${network_name}") {
             sh '''
                 cd vcx/dummy-cloud-agent
-                sed -i 's/\\("protocol_type": "\\)1.0/\\12.0/' config/sample-config.json
                 LIBRARY_PATH=./ LD_LIBRARY_PATH=./:${LD_LIBRARY_PATH} RUST_BACKTRACE=1 RUST_LOG=indy=info cargo run config/sample-config.json &
```

Comment:
You either need to move running 2.0 tests before 1.0 or update config/sample-config.json again to return 1.0 protocol version. `testEnv.inside` keeps folder changes here.

Reply:
Seems like you've pushed the fix for me, anything on my side I can do? By the way, seems like jenkins pipeline is no longer public - when I click `details` on CI run, it displays Jenkins login page.